docs/pages/pmd/userdocs/help.md

```diff
@@ -1,8 +1,26 @@
 ---
 title: Getting Help
-sidebar: pmd_sidebar
 permalink: pmd_userdocs_help.html
-folder: pmd/userdocs
+author: Andreas Dangel <andreas.dangel@adangel.org>
+last_updated: September 2017
 ---
 
-Asking questions, reporting bugs, ...
+There are numerous ways of getting help:
+
+*   Browse already existing questions on [StackOverflow](https://stackoverflow.com/questions/tagged/pmd),
+    that are [tagged with `pmd`](https://stackoverflow.com/questions/tagged/pmd).
+
+*   You can search on [StackOverflow](https://stackoverflow.com/) for a similar problem or ask a new question there.
+
+    Don't forget to tag the question with `pmd`.
+
+*   Create a new [github issue](https://github.com/pmd/pmd/issues) with your question.
```

Comment:
I feel we shouldn't encourage this, even if we do answer to such issues... it would split knowledge (questions answered there vs here), and introduce noise to our issue tracker.

Reply:
You're right, I'll remove this.